package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "saucectl-run-action",
+  "version": "0.1.0",
+  "description": "A GitHub actions providing control of saucectl",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "ncc build src/index.js -o dist/main",
+    "test": "npx jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/saucectl/saucectl-run-action.git"
+  },
+  "author": "felix@saucelabs.com",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/saucectl/saucectl-run-action/issues"
+  },
+  "jest": {
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "src/**/*.js"
+    ]
+  },
```

Comment:
Let's move this into a dedicated `jest.config.js`. Also it would be great to define a coverage treshold so we ensure that test coverage never decreases.